teachopencadd/talktorials/T015_protein_ligand_docking/talktorial.ipynb

```diff
@@ -4,231 +4,206 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">for atom in molecule.atoms:
&nbsp; &nbsp; &nbsp; &nbsp; atom.partialcharge
</pre><p> </p><p>What is this block doing? I see no assignment or method. Is this a weird property with side effects?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>yes, weird property with side effects, I found a way to change the calculation of partial charges to be more clear</p>